### DISCUSSION TITLE
fluentd log rotation， daily， how many log files retained？

### DISCUSSION THREAD

**Discussion by nuaazt** [fluent/fluentd#3716] [Q&A]
hello， I want to output the log to files and I want to get one log file daily.  Also, when the number of files exceeds 10, delete the oldest one. 
I config the comand option :
fluentd -o /path/to/log_file --log-rotate-age daiy. 

But how can it do the "when the number of files exceeds 10, delete the oldest one"？

**Comment by kenhys** [comment]
Combination with using /etc/logrotate.d/ configuration will work for you.
